### PR TITLE
core: add help message

### PR DESCRIPTION
**Pull request by LOSEARDES77**
Add a -help arg option to solve #69

### CODE REVIEW

**Review by fufexan — COMMENTED** [review]
Perhaps make a function, to mimic [the code in Hyprland](https://github.com/hyprwm/Hyprland/blob/main/src/main.cpp#L14-L20).

**Comment by vaxerski** [comment]
ye

File: src/main.cpp [hunk 1]
```diff
@@ -14,10 +22,14 @@ int main(int argc, char** argv, char** envp) {
         else if (arg == "--quiet" || arg == "-q")
             Debug::quiet = true;
 
-        else if (arg == "--display" && i + 1 < argc) {
+        else if ((arg == "--display" || arg == "-d" ) && i + 1 < argc) {
```
Comment (vaxerski) [inline src/main.cpp:25]: don't add a short opt for display.

File: src/main.cpp [hunk 2]
```diff
@@ -2,6 +2,14 @@
 #include "config/ConfigManager.hpp"
 #include "core/hyprlock.hpp"
 
+void help(){
+    std::cout <<    "Usage: hyprlock [options]\n\n"
+                    "Options:\n"
+                    "  -v, --verbose            - Enable verbose logging\n"
+                    "  -q, --quiet              - Disable logging\n"
+                    "  --display (display)  - Specify the Wayland display to connect to\n"
```
Comment (vaxerski) [inline src/main.cpp:10]: misaligned :(

**Review by vaxerski — APPROVED** [review]
thanks :)

**Pull request merged**